src/driver-sinowealth.c

```diff
@@ -137,19 +138,39 @@ _Static_assert(sizeof(struct sinowealth_config_report) == SINOWEALTH_CONFIG_SIZE
 struct sinowealth_data {
 	/* this is kinda unnecessary at this time, but all the other drivers do it too ;) */
 	struct sinowealth_config_report config;
+	bool is_sensor_3360;
```

Comment:
A comment here about the meaning of `!is_sensor_3360` might be good

Reply:
Maybe if we make it an enum sensor instead it would be more obvious?

Reply:
I pushed a commit that changes this to an enum. Does that make it clearer?

Reply:
Very neat! :)